content/hi/docs/reference/glossary/workload.md

```diff
@@ -0,0 +1,19 @@
+---
+title: काम का बोझ
+id: workloads
+date: 2019-02-13
+full_link: /docs/concepts/workloads/
+short_description: >
+   कार्यभार एक ऐप्लिकेशन है जो Kubernetes पर चल रहा है।
+
+aka: 
+tags:
+- मौलिक
+---
+   कार्यभार एक ऐप्लिकेशन है जो Kubernetes पर चल रहा है।
+
+<!--more--> 
+
+विभिन्न मूल ऑब्जेक्ट जैसे DaemonSet(डेमन सेट), Deployment(डिप्लॉयमेंट), Job(जॉब), ReplicaSet(रिप्लिका सेट) और StatefulSet(स्टेटफुल सेट) कार्यभार के विभिन्न प्रकार या भागों को प्रतिनिधित करती हैं।
```

Comment:
You do not need to add the English words along with it. Also, DaemonSet & Deployment are spelled wrong. Please refer to the Hindi documentation to see how they're correctly spelled.

Reply:
विभिन्न मूल ऑब्जेक्ट जैसे डाइमनसेट, डिप्लायमेंट,जॉब, रिप्लिका सेट और स्टेटफुल सेट कार्यभार के विभिन्न प्रकार या भागों को प्रतिनिधित करती हैं।
how about this ma'am ? @divya-mohan0209